zerver/openapi/zulip.yaml

```diff
@@ -18742,25 +18742,48 @@ paths:
                   type: string
                   example: The marketing team.
                 can_mention_group:
-                  allOf:
-                    - description: |
-                        Either the ID of a named user group that has permission to
-                        mention the group, or an object describing the set of users
-                        and groups who have permission mention the group.
+                  description: |
+                    The set of users who have permission to mention this group.
 
-                        This setting cannot be set to `"role:internet"` and `"role:owners"`
-                        system groups.
+                    This parameter is expressed as a [group-setting update](/api/updating-group-settings).
+                    See [details](/api/updating-group-settings) on that format.
 
-                        **Changes**: Before Zulip 9.0 (feature level 258), the
-                        `can_mention_group` field was always an integer.
+                    This setting cannot be set to `"role:internet"` and `"role:owners"`
+                    system groups.
 
-                        **Changes**: Before Zulip 8.0 (feature level 198),
-                        the `can_mention_group` setting was named `can_mention_group_id`.
+                    **Changes**: In Zulip 9.0 (feature level 260), this was updated
+                    to only accept an object with `old` and `new` fields.
 
-                        New in Zulip 8.0 (feature level 191). Previously, groups
-                        could be mentioned if and only if they were not system groups.
-                    - $ref: "#/components/schemas/CanMentionGroup"
-                  example: 12
+                    **Changes**: Before Zulip 9.0 (feature level 258), the
+                    `can_mention_group` field was always an integer.
+
+                    **Changes**: Before Zulip 8.0 (feature level 198),
+                    the `can_mention_group` setting was named `can_mention_group_id`.
+
+                    New in Zulip 8.0 (feature level 191). Previously, groups
+                    could be mentioned if and only if they were not system groups.
+                  type: object
+                  additionalProperties: false
+                  properties:
+                    new:
+                      allOf:
+                        - description: |
+                            The new value for who would have the permission to
+                            mention the group.
+                        - $ref: "#/components/schemas/CanMentionGroup"
+                    old:
+                      allOf:
+                        - description: |
+                            The expected current value for who has the permission
+                            to mention the group.
+                        - $ref: "#/components/schemas/CanMentionGroup"
```

Comment:
Screenshot for reference -
![image](https://github.com/zulip/zulip/assets/35494118/3c71b84d-3d99-40c9-ad30-8ff624eabc4c)

Reply:
Looks great!